CONTRIBUTING.md

```diff
@@ -0,0 +1,102 @@
+# Contributing
+ 
+Thank you for considering contributing to Symfony UX! 
+
+Symfony UX is an open source, community-driven project, and we are happy to receive contributions from the community!
+
+> [!TIP]
+> It's a good idea to read the [Symfony's Contribution Guide](https://symfony.com/doc/current/contributing/index.html) first, even if not all of it applies to Symfony UX and should be adapted to this project (e.g.: Symfony UX has only one base branch, `2.x`).
+
+## Reporting an issue
+
+If you either find a bug, have a feature request, or need help/have a question, please [open an issue](https://github.com/symfony/ux/issues/new/choose).
+
+Please follow the issue template and provide as much information as possible,
+and remember to follow our [Code of Conduct](https://symfony.com/doc/current/contributing/code_of_conduct/index.html)
+as well, to ensure a friendly environment for all contributors.
+
+## Contributing to the code and documentation
+
+Thanks for your interest in contributing to Symfony UX! Here are some guidelines to help you get started.
+
+### Forking the repository
+
+To contribute to Symfony UX, you need to fork the repository on GitHub. This will give you a copy of the code under your GitHub user account.
```

Comment:
```suggestion
To contribute to Symfony UX, you need to [fork the **symfony/ux** repository](https://github.com/symfony/ux/fork) on GitHub. This will give you a copy of the code under your GitHub user account. 
```

Maybe a link here ? https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo